src/mongocxx/options/find.cpp

```diff
@@ -31,10 +31,18 @@ void find::comment(std::string comment) {
     _comment = comment;
 }
 
+void find::cursor_type(enum cursor_type cursor_type) {
```

Comment:
Can you put all of the new getters and setters that aren't related to MaxAwaitTimeMs into a separate commit?